### PR TITLE
Fixes gh-1650 Add columns to existing tables in SQL

### PR DESCRIPTION
**Pull request by pgulutzan**
Warning: not just an .rst change, also an .svg and .spec change.

### CODE REVIEW

File: doc/reference/reference_sql/sql_statements_and_clauses.rst [hunk 1]
```diff
@@ -85,6 +85,13 @@ exist. Example: |br|
 ``-- renaming a table:``
 ``ALTER TABLE t1 RENAME TO t2;``
 
+For adding a :ref:`column <sql_column_def>` with ``ADD COLUMN``,
+the table must exist, the table must be empty,
+the column name must be unique within the table.
+Example with a STRING column that must start with X: |br|
+``ALTER TABLE t1 ADD COLUMN s4 STRING CHECK (s4 LIKE 'X%');`` |br|
+ALTER TABLE ... ADD COLUMN support was added in version :doc:`2.7.1 </release/2.7.1>`.
```
Comment (NickVolynkin) [inline doc/reference/reference_sql/sql_statements_and_clauses.rst:93]: Thanks for mentioning the version!

**Review by NickVolynkin — APPROVED** [review]

**Comment by NickVolynkin** [comment]
@pgulutzan Peter, thanks! 

There's new deployment magic that I'd like to tell you about. Now we can build the changes in a pull request and preview them on the test website.

To deploy a pull request, one should add a `build-branch` label to the pull request, like I've done here. Then GitHub will build and deploy the current commit, and then do the same for each new commit in this PR.

While the build is in progress, pull request will have this banner:

изображение

When it's ready, it'll change to this:

Now one can click the `details` button and then open the link to the deployment:

 
I've added a link back from the release notes and formatted the code example a little. This commit was already built and deployed, you can have a look.

If you're okay with my additions, please merge the pull request.

**Comment by veod32** [comment]
I added another link back from the release notes page. Now we are good to go with merging.

**Comment by pgulutzan** [comment]
@NickVolynkin: When you say I should add a "build-branch" label, I guess you mean that, for example on this pull request, I should click on "Labels" and then click the branch number, which is 2.7.1 (somebody clicked "deploy-branch" which I guess is not what I should do), okay. I have explained why I believe that adding "code-block:: sql" is generally a bad idea but won't argue about this case. On prior occasions I used the command line to merge, but this time I just clicked the merge button on this site. It seems that I was supposed to put something in the release notes pointing to ALTER TABLE, okay.

**Comment by pgulutzan** [comment]
@NickVolynkin: I overlooked two things:
1. In the list after the word "Syntax:" I should have added ALTER TABLE table-name ADD COLUMN column-name column-definition;".
2. In the Limitations I should have removed "It is not possible to add or drop a column. However, there is a plan to make it possible to add a column for an empty table (Issue#3075)." Thanks to @veod32 for noticing this.

If you do not object, I will simply fix those things and push.

**Comment by NickVolynkin** [comment]
@pgulutzan please do!

**Pull request merged**